test/application/rooms.js

```diff
@@ -0,0 +1,25 @@
+var assert = require('assert');
+var room;
+
+describe('Rooms features', function () {
+	describe('Rooms', function () {
+		describe('Rooms.get', function () {
+			it('should be a function', function () {
```

Comment:
ewwwwwwww, a function named `it`...

Honestly, at least half the reason I haven't gotten around to adding tests is because most of the JavaScript test suites do things like this to make the code read more English-like.

I prefer my JavaScript to look like JavaScript, not like English, thank you very much. <_<